taxi/urls.py

```diff
@@ -37,14 +41,60 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/",
+        CarListView.as_view(),
+        name="car-list"
+    ),
+    path(
+        "cars/<int:pk>/",
+        CarDetailView.as_view(),
+        name="car-detail"
+    ),
+    path(
+        "cars/create/",
+        CarCreateView.as_view(),
+        name="car-create"
+    ),
+    path(
+        "cars/<int:pk>/update/",
+        CarUpdateView.as_view(),
+        name="car-update"
+    ),
+    path(
+        "cars/<int:pk>/delete/",
+        CarDeleteView.as_view(),
+        name="car-delete"
+    ),
+    path(
+        "cars/<int:pk>/reassignment",
+        add_delete_driver,
+        name="car-add_delete"
```

Comment:
Don't mess `-` and `_` symbols in names. It is strange that you have `car delete` and `car add delete` paths